rpm/get_version_string.sh

```diff
@@ -1,25 +1,26 @@
 #!/bin/bash
 
-# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+# Copyright 2014 Telefonica Investigacion y Desarrollo, S.A.U
 #
-# This file is part of perseo-core.
+# This file is part of perseo-fe.
```

Comment:
This repo is perseo-core, not fe...

In general, I think that changes from L1 to L24 should be rolled-back.